src/main/fc/settings.c

```diff
@@ -626,13 +632,20 @@ const clivalue_t valueTable[] = {
 
     { "osd_rssi_alarm",             VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 100 }, PG_OSD_CONFIG, offsetof(osdConfig_t, rssi_alarm) },
     { "osd_cap_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 20000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, cap_alarm) },
-    { "osd_time_alarm",             VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 60 }, PG_OSD_CONFIG, offsetof(osdConfig_t, time_alarm) },
     { "osd_alt_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 10000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, alt_alarm) },
 
+    { "osd_tim1_type",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OSD_TIMER_NAMES }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_1].type) },
+    { "osd_tim1_prec",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OSD_TIMER_PRECISION }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_1].precision) },
+    { "osd_tim1_alarm",             VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 60 }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_1].alarm) },
+
+    { "osd_tim2_type",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OSD_TIMER_NAMES }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_2].type) },
```

Comment:
I think `TABLE_OSD_TIMER_NAMES` should be `TABLE_OSD_TIMER_TYPES` for consistency.